cmd/opm/registry/add.go

```diff
@@ -0,0 +1,81 @@
+package registry
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/sqlite"
+)
+
+func newRegistryAddCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "add",
+		Short: "add operator bundle to operator registry DB",
+		Long:  `add operator bundle to operator registry DB`,
+
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if debug, _ := cmd.Flags().GetBool("debug"); debug {
+				logrus.SetLevel(logrus.DebugLevel)
+			}
+			return nil
+		},
+
+		RunE: addFunc,
+	}
+
+	rootCmd.Flags().Bool("debug", false, "enable debug logging")
+	rootCmd.Flags().StringP("database", "d", "bundles.db", "relative path to database file")
+	rootCmd.Flags().StringSliceP("bundle-images", "b", []string{}, "comma separated list of links to bundle image")
```

Comment:
non-blocking: did we think about making these positional? 

```
opm add -d bundles.db quay.io/my/manifests:1 quay.io/my/manifests:2 
```

Reply:
That looks cool